docs/site/docusaurus.config.js

```diff
@@ -149,7 +148,25 @@ const config = {
         },
       },
     ],
-    'plugin-image-zoom'
+    'plugin-image-zoom',
+    [
+      'docusaurus-plugin-openapi-docs',
+      {
+        id: 'openapi',
+        docsPluginId: 'classic',
+        config: {
+          coreApiV2: {
+            specPath:
```

Comment:
Wouldn't it be better to use a GH URL?

Reply:
Yes. I can change that now. But when I created this PR the repo wasn't public yet. Will change it 👍

Reply:
Done